plugins/kubernetes/app/models/kubernetes/role_verifier.rb

```diff
@@ -0,0 +1,106 @@
+module Kubernetes
+  class RoleVerifier
+    DEPLOYISH = ['Deployment', 'DaemonSet'].freeze
+
+    def initialize(role)
```

Comment:
role_definition ... the content of the kubernetes/xyz.yml

On Thu, Jun 9, 2016 at 1:26 PM, Jon Moter <notifications@github.com> wrote:

> In plugins/kubernetes/app/models/kubernetes/role_verifier.rb
> <https://github.com/zendesk/samson/pull/1053#discussion_r66515147>:
>
> > @@ -0,0 +1,106 @@
> > +module Kubernetes
> > +  class RoleVerifier
> > +    DEPLOYISH = ['Deployment', 'DaemonSet'].freeze
> > +
> > +    def initialize(role)
>
> Not clear what role is supposed to be here. I would've assumed a
> Kubernetes::Role object. But it's actually supposed to be a string, right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1053/files/0334af0171d0cb13ab252d1f5eb73ba2aae1ed1c#r66515147>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ4fpuAOtakGRTmvPapiqbA7VOxhOks5qKHbagaJpZM4Ixhgp>
> .
>